hazelcast-jet-all/src/test/java/com/hazelcast/jet/server/JetCommandLineTest.java

```diff
@@ -92,10 +100,13 @@ public static void afterClass() {
     public void before() {
         JetConfig cfg = new JetConfig();
         cfg.getHazelcastConfig().addEventJournalConfig(new EventJournalConfig().setMapName(SOURCE_NAME));
+        cfg.getHazelcastConfig().getNetworkConfig().setPort(9000);
```

Comment:
will this cause a problem with parallel tests? maybe better to change group name?